roles/container-engine/cri-o/molecule/default/molecule.yml

```
@@ -26,7 +26,7 @@ platforms:
     groups:
       - kube-master
   - name: fedora
-    box: fedora/31-cloud-base
+    box: fedora/33-cloud-base
```

Technically this can be fixed pointing to the archive mirror but I prefer what you're proposing.

```suggestion
    box: fedora/31-cloud-base
    box_url: https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/31/Cloud/x86_64/images/Fedora-Cloud-Base-Vagrant-31-1.9.x86_64.vagrant-libvirt.box
```

---

Thanks. Didn't know where the archived images were.
but it seems to me that for the test it is better to use the current versions